src/racing/actors.c

```diff
@@ -2517,121 +1426,121 @@ s16 addActorToEmptySlot(Vec3f pos, Vec3s rot, Vec3f velocity, s16 actorType) {
     return -1;
 }
 
-s16 func_8029ED38(Vec3f pos, s16 actorType) {
+s16 allocate_actor_with_new_position(Vec3f pos, s16 actorType) {
     Vec3f vel;
     Vec3s rot;
 
     vec3f_set(vel, 0.0f, 0.0f, 0.0f);
     vec3s_set(rot, 0, 0, 0);
-    return addActorToEmptySlot(pos, rot, vel, actorType);
+    return add_actor_to_empty_slot(pos, rot, vel, actorType);
 }
 
-// I swear we have a struct that looks like this already but I cannot find it anywhere
+// not ActorSpawnData but very similar in structure and use
 struct test {
     Vec3s thing;
 };
 
-UNUSED void func_8029ED98(Player *player, uintptr_t arg1) {
+UNUSED void proto_loading_actor_spawn_data(Player *player, uintptr_t arg1) {
     Vec3f sp64;
     struct test *var_s0;
     s32 segment = SEGMENT_NUMBER2(arg1);
     s32 offset = SEGMENT_OFFSET(arg1);
 
     var_s0 = (struct test *) VIRTUAL_TO_PHYSICAL2(gSegmentTable[segment] + offset);
-    while (var_s0->thing[0] != -0x8000) {
+    while (var_s0->thing[0] != END_OF_SPAWN_DATA) {
         sp64[0] = var_s0->thing[0] * gCourseDirection;
         sp64[1] = var_s0->thing[1];
         sp64[2] = var_s0->thing[2];
         if(arg1 & arg1){}
-        func_8029EEB8(player, sp64, 5.0f, 40.0f, 0.8f);
+        is_collide_and_apply(player, sp64, 5.0f, 40.0f, 0.8f);
         var_s0++;
     }
 }
 
-s32 func_8029EEB8(Player *player, Vec3f pos, f32 arg2, f32 arg3, f32 arg4) {
-    f32 temp_f0_2;
-    f32 temp_f0_3;
-    f32 temp_f14;
-    f32 temp_f16;
-    f32 temp_f18;
+bool is_collide_and_apply(Player *player, Vec3f pos, f32 minDist, f32 dist, f32 arg4) {
```

Comment:
is_colliding_in_bounds
or
is_colliding_within_bounds

Reply:
I think it's good to keep the information of this function apply the collision with the player and not only detect it so
is_colliding_in_bounds_and_apply_player_collision_to_it

Reply:
I'm fine with whatever. The main thing I was looking at here is that `is_collide_` isn't proper grammar. but `is_colliding` is. Or `is_collision_` (depending on context)

Reply:
Name this function:

`resolve_collision_box_vs_sphere_aabb`

Reply:
it's more two spherer then a box and a shere

Reply:
I finally rename it is_colliding_and_resolve

Reply:
query_resolve_collision_player_object ?

Rename TRUE/FALSE to COLLISION and NO_COLLISION and update the if (query() == 1) to `if (query() == COLLISION)`